test/failure_handler/src/share/classes/jdk/test/failurehandler/Utils.java

```diff
@@ -66,6 +69,16 @@ public static Properties getProperties(String name) {
         String resourceName = String.format(
                 "/%s.%s", name.toLowerCase(), "properties");
         InputStream stream = Utils.class.getResourceAsStream(resourceName);
+
+        // EFH_CONF_DIR might re-defined to load custom configs for development purposes
```

Comment:
this also seems to be unrelated to the subject and does require documentation in `test/failure_handler/README`.